modulemd/v2/tests/test_data/dependson_v2.yaml

```
@@ -13,9 +13,12 @@ data:
     - MIT
   dependencies:
   - buildrequires:
-      platform: [ f30 ]
+      platform: [f30]
```

Please don't remove these spaces. They make it more readable.
```suggestion
      platform: [ f30 ]
```